### PR TITLE
QLSWG: Add Monarch interview answers.

### PR DESCRIPTION
**Pull request by manolama**

### CODE REVIEW

File: working-groups/query-standardization/dsl-survey/google-monitoring-query-language/README.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+1. **Are there any particular strengths or weaknesses in the language after observing it in use?**:
+
+   * **Strengths**:
+     
+     Even though time series data models have a somewhat steep learning curve, we observed that exposing the 
+     data model in its entirety gives users a level of power and freedom that they ultimately come to appreciate. 
+     Hiding or glossing over features of the data model can lead to surprises that users find frustrating.
+
+     Most of the other strengths are downstream from this first strength, e.g. Monarch's early engineers made
```
Comment (vjsamuel) [inline working-groups/query-standardization/dsl-survey/google-monitoring-query-language/README.md:49]: would be nice if we can get some examples

File: working-groups/query-standardization/dsl-survey/google-monitoring-query-language/README.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+     data model in its entirety gives users a level of power and freedom that they ultimately come to appreciate. 
+     Hiding or glossing over features of the data model can lead to surprises that users find frustrating.
+
+     Most of the other strengths are downstream from this first strength, e.g. Monarch's early engineers made 
+     many good decisions about their data model, so exposing that data model directly to users allows them to 
+     benefit directly from those good decisions.
+
+   * **Weaknesses**: 
+
+     MQL achieves terse syntax through aggressive syntactic sugar. There are often a dozen or more ways to express
```
Comment (vjsamuel) [inline working-groups/query-standardization/dsl-survey/google-monitoring-query-language/README.md:55]: would be nice if we can get some examples

**Review by vjsamuel — APPROVED** [review]

**Comment by manolama** [comment]
Merging for now

File: working-groups/query-standardization/dsl-survey/google-monitoring-query-language/README.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+1. **Does the DSL allow for writing data or configuring the backing store or is it for querying only?**:
+
+   The language itself only supports querying, but Monarch is able to write query results back to the data store; this 
+   cannot be done on an ad-hoc basis and needs to be configured. MQL has no support for modifying configuration.
+
+## References
+
+* [MQL Overview](https://cloud.google.com/monitoring/mql)
+* [About the MQL Language](https://cloud.google.com/monitoring/mql/query-language)
+* [Monarc: Google’s Planet-Scale In-Memory Time Series Databaseh](https://www.vldb.org/pvldb/vol13/p3181-adams.pdf)
```
Comment (bernot-dev) [inline working-groups/query-standardization/dsl-survey/google-monitoring-query-language/README.md:112]: Small typo:
Monarc**h**: Google’s Planet-Scale In-Memory Time Series Database~~h~~

**Pull request merged**